apps/builder/app/builder/features/style-panel/style-source/use-sortable.tsx

```diff
@@ -81,10 +94,24 @@ export const useSortable = <Item extends { id: string }>({
         return false;
       }
 
-      return getItemId(closest) || false;
+      const itemId = getItemId(closest);
+      if (itemId === undefined) {
+        return false;
+      }
+
+      return itemId;
     },
-    onStart({ data }) {
-      setDragItemId(data);
+    onStart({ data: itemId }) {
+      const item = items.find((item) => item.id === itemId);
+      if (items.at(-1) === item && item?.source === "local") {
+        toast.error(
+          "Local style source should be last and cannot be reordered"
```

Comment:
@taylornowotny this wording ok? its shown when you try to reorder local

Reply:
Good idea @TrySound , wording can improve to: "Local token is always last and can not be moved"